src/Sarif.Viewer.VisualStudio/Views/CallTreeView.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+namespace Microsoft.Sarif.Viewer.Views
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using System.Windows;
+    using System.Windows.Controls;
+    using Microsoft.Sarif.Viewer.Models;
+
+    /// <summary>
+    /// Implementation of a tree view for the call tree.
+    /// </summary>
+    /// <remarks>
+    /// The selection in the tree view is maintained by the data model through the <see cref="CallTree.SelectedItem"/> property.
+    /// What this class is ensuring that when the selected item property is modified in the data model 
```

Comment:
> What this class is ensuring [](start = 7, length = 28)

This class ensures that...

Reply:
Add commas as well..

---
In reply to: [503386481](https://github.com/microsoft/sarif-visualstudio-extension/pull/214#discussion_r503386481) [](ancestors = 503386481)